psamm/database.py

```diff
@@ -164,28 +164,67 @@ class DictDatabase(MetabolicDatabase):
 
     def __init__(self):
         super(DictDatabase, self).__init__()
+        self._model_name = None
+        self._biomass_reaction = None
+        self._default_flux_limit = None
+        self._extracellular_compartment = None
+        # _flux_limits is a map from reaction_id to (upper, lower)
+        self._flux_limits = {}
+        # _medium is a map from reaction_id to (Compound, upper, lower)
+        self._medium = {}
+        # reaction IDs in the model
+        self._model = set()
+        # compound map from compound ID to CompoundEntry
+        self._compound_entries = defaultdict(dict)
+        # reaction map from reaction ID to ReactionEntry
+        self._reaction_entries = defaultdict(dict)
+        # original attributes in DictDatabase, consider to be moved
         self._reactions = defaultdict(dict)
```

Comment:
How about change this attribute name to _reaction_values in the future now matter where it would be? Also for the related code using this attribute.